cmd/guard-service/services.go

```diff
@@ -49,10 +49,16 @@ func serviceKey(ns string, sid string, cmFlag bool) string {
 	return service
 }
 
-func newServices() *services {
+func _newServices() *services {
```

Comment:
Hmm, wondering why using _ in the name... I know it's private but Golang has strong formatting rules and people expect you to follow them. At a minimum add a comment to explain your reasoning.